### ISSUE TITLE
intent_loss and slot_loss

### ISSUE THREAD

**Issue by guliqi** [yuanxiaosc/BERT-for-Sequence-Labeling-and-Text-Classification#27]
why intent loss equals to reduce MEAN of intent_per_example_loss but slot loss equals to reduce SUM of slot_per_example_loss?

**Comment by yuanxiaosc** [comment]
@guliqi 
Because in code design, I subjectively think that the two tasks are of the same importance, as shown in the loss function: loss = intent_ loss + slot_ Loss, you can add weight coefficient to them to change the focus on tasks during model training, such as loss = 0.8 * intent_ loss + 0.2*slot_ loss。
